### ISSUE TITLE
GAP.prompt() broken in ubuntu docker container

### ISSUE THREAD

**Issue by mohamed-barakat** [oscar-system/GAP.jl#465]
```
julia> GAP.prompt()
gap> gap> 1+1;
2
gap> gap>
```
After pressing `^L` one prompt vanishes but the double prompt reappears again after next ENTER.

**Comment by fingolfin** [comment]
Sounds like another ncurses/readline issue? But you don't provide enough information to reproduce or debug this, really. E.g. Is this with GAP.jl 0.4.0 or master? Which Ubuntu version, which Julia? Can you perhaps specify which exact Docker image you are using? Etc.

**Comment by mohamed-barakat** [comment]
1. `ubuntu:latest`
```
$ docker run -it ubuntu /bin/bash
Unable to find image 'ubuntu:latest' locally
latest: Pulling from library/ubuntu
d51af753c3d3: Pull complete
fc878cd0a91c: Pull complete
6154df8ff988: Pull complete
fee5db0ff82f: Pull complete
Digest: sha256:747d2dbbaaee995098c9792d99bd333c6783ce56150d1b11e333bbceed5c54d7
Status: Downloaded newer image for ubuntu:latest
```
2. `apt-get update; apt-get install -y build-essential curl julia libjulia-dev` # this will get you julia 1.4.1
3. `pkg> dev GAP`

**Comment by fingolfin** [comment]
And what does `printenv TERM` say in that VM?

**Comment by fingolfin** [comment]
OK, I replicated this; `TERM` is set to `xterm` for me.

So I am pretty sure this is an artifact of running into a barebones container. If you look at the Julia and GAP banners, you will notice that they are "degraded" when compared to how they usually look: for Julia, I see `Ubuntu ⛬  julia/1.4.1+dfsg-1` (note the broken character after "Ubuntu"). For GAP the "box" around GAP is not rendered with graphics. The "double prompt" actually is an artifact of terminal escape codes which are not handled correctly (try pressing "space" then "backspace"; the prompt is "fixed" by this).

That said, I tried `apt install gap-core` (BTW, for some reasons this installs TeXLive, X11 and more... I emailed Bill to ask why). The `gap` installed this way works fine... So it may still be an issue of ncurses not finding the right terminfo (but then why does backspace work fine) or something like that... Hrmmm.
